bin/bats

```diff
@@ -3,42 +3,70 @@
 set -e
 
 BATS_READLINK='true'
+BATS_REALPATH=''
 if command -v 'greadlink' >/dev/null; then
   BATS_READLINK='greadlink'
 elif command -v 'readlink' >/dev/null; then
   BATS_READLINK='readlink'
 fi
 
+set +e
+$BATS_READLINK -m $0 &>/dev/null;
+[ $? -ne 0 ] && BATS_REALPATH='realpath'
+set -e
+
+resolve_path() {
+  local path=$1
+  local target_path;
+
+  if [[ -L $path ]]; then
+    # Busybox has no readlink from coreutisl
+    if [[ ! -z "$BATS_REALPATH" ]]; then
+      target_path="$(realpath "$path")"
```

Comment:
Should this be 

```bash
target_path="$(${BATS_REALPATH} "$path")"
```

?

Reply:
I don't think so, but I'll have a look at the logic when I rework the MR.